ckanext/datajson/package2pod.py

```diff
@@ -146,7 +146,12 @@ def export_map_fields(package, json_export_map, redaction_enabled=False):
                     method = getattr(Wrappers, wrapper)
                     if method:
                         Wrappers.current_field_map = field_map
-                        dataset[key] = method(dataset.get(key))
+                        try:
```

Comment:
would you mind (if possible) writing a unit test for this exception? just in case we refactor this in the future.

Reply:
I've added a unit test for a dataset with a resource missing a url.